inc/thorvg.h

```diff
@@ -317,9 +324,9 @@ class TVG_EXPORT Fill
      * @brief Sets the parameters of the colors of the gradient and their position.
      *
      * @param[in] colorStops An array of ColorStop data structure.
-     * @param[in] cnt The count of the @p colorStops array equal to the colors number used in the gradient.
+     * @param[in] cnt The size of the @p colorStops array equal to the colors number used in the gradient.
```

Comment:
count is more clear. its parameter says "count"
size is ambiguous since  data size * data count

Reply:
@mgrudzinska then it'd rather this:
The count of the @p colorStop equal ... 
I think size can't be applied this. it requires exact number of the element. not size.

Reply:
restored your version